rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -40,6 +36,8 @@
  * @author Oscar Guindzberg
  */
 public class BridgeStorageProvider {
+    public static final Coin STARTING_FEE_PER_KB = Coin.MILLICOIN;
```

Comment:
As discussed, this should be a constant on `BridgeConstants`

Reply:
Never heard that, but it sounds great